README.md

```diff
@@ -118,10 +131,11 @@ python dense_retriever.py \
 	--ctx_file  {path to all documents .tsv file} \
 	--qa_file {path to test|dev .csv file} \
 	--encoded_ctx_file "{encoded document files glob expression}" \
-	--out_file {path to output json file with results}
+	--out_file {path to output json file with results} \
+  --n-docs 200
```

Comment:
We didn't always used 200. It was 100 sometimes. It the deafult valu in not changed in the code, may be put some comments here about choosing among 100|200 options.

Reply:
I thought you used 200 for the provided preprocessed data?